install/mysql/glpi-empty.sql

```diff
@@ -2739,6 +2739,7 @@ CREATE TABLE `glpi_entities` (
   `mail_domain` varchar(255) DEFAULT NULL,
   `entity_ldapfilter` text,
   `mailing_signature` text,
+  `url_base` text,
```

Comment:
Dow e really need a `TEXT` for a base URL? I have in mind many URLs may have way more than 255 characters; but for a base URL, I'm not sure this is usefull. `mail_domain` is set tu `varchar(255)`; I guess `url_base` may be aswell.
Anyways, not a blocker.